instrumentation/jedis-6.0.0/src/main/java/redis/clients/jedis/JedisPubSubBase_Instrumentation.java

```diff
@@ -0,0 +1,82 @@
+/*
+ *
+ *  * Copyright 2025 New Relic Corporation. All rights reserved.
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package redis.clients.jedis;
+
+import com.newrelic.agent.bridge.datastore.DatastoreVendor;
+import com.newrelic.api.agent.DatastoreParameters;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.Trace;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+
+@SuppressWarnings({ "ResultOfMethodCallIgnored", "unused" })
+@Weave(type = MatchType.ExactClass, originalName = "redis.clients.jedis.JedisPubSubBase")
+public class JedisPubSubBase_Instrumentation {
+
+  private final JedisSafeAuthenticator authenticator = Weaver.callOriginal();
+
+  public final void proceed(Connection client, Object... channels) {
```

Comment:
I might be missing something obvious, but do the `proceed` and `proceedWithPatterns` method need to be weaved? I don't see any side effects from simply calling `Weaver.callOriginal()`.

Reply:
Good point, it's been that way since jedis-4.0.0 and might have no use anymore. I'll look back through the history and see if I can find some context.